schedule/functional/extra_tests_transactional_server.yaml

```diff
@@ -9,6 +9,7 @@ schedule:
     - update/zypper_clear_repos
     - console/zypper_ar
     - console/zypper_ref
+    - console/java
```

Comment:
Schedule the java test, after `console/update_alternatives`

Reply:
okay, updated with new test run, thanks